lib/stock_picker.rb

```diff
@@ -0,0 +1,15 @@
+def stock_picker(prices)
+  profit = 0
```

Comment:
raise an error if the argument is not `Array` and if the length of prices array is less than 2